pkg/policy/portrange.go

```diff
@@ -0,0 +1,99 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Cilium
+
+package policy
+
+import (
+	"fmt"
+	"math"
+	"math/bits"
+)
+
+// MaskedPort is a port with a wild card mask value
```

Comment:
nit: When clicking on `portrange.go`, I would expect the first structure to be `PortRange`  and not `MaskedPort`

Reply:
The port range is represented by a masked port. We need to use masks for policy Keys (rather than have a more semantically convenient type that is human readable) because that is what ends up in the datapath.

Reply:
Thanks! Maybe worth mentioning it 🤷🏽‍♀️